vlsi/example-tools.yml

```diff
@@ -0,0 +1,24 @@
+# SRAM Compiler compiler options
+vlsi.core.sram_generator_tool: "sram_compiler"
+# You should specify a location for the SRAM generator in the tech plugin
+vlsi.core.sram_generator_tool_path: []
```

Comment:
If this is used in the asap7 demo, a more explicit (but technically redundant, since hammer knows about it already) path would be:
`vlsi.core.sram_generator_tool_path: ["/home/ff/eecs151/hammer/src/hammer-vlsi/technology/asap7"]`

I don't know if you want to give this option for users who don't desire to use barstools' macrocompiler:
```
# Uncomment these if you are not using macrocompiler
# vlsi.inputs.sram_parameters: "/home/ff/eecs151/hammer/src/hammer-vlsi/technology/asap7/sram-cache.json"
# vlsi.inputs.sram_parameters_meta: ["transclude", "json2list"] 
```

Reply:
As previously noted, it is meant to be used as a template for a general case, rather than solving a single-case problem. 
Given that assumption, I think any path that includes the Berkeley instructional machines eecs151 installation directory would be confusing.

Reply:
Oh sorry, I forgot to take out `/home/ff/eecs151/`! Edited above.

Reply:
Why would I want to specify the explicit path if Hammer is already aware if it through the tech plugin?

Reply:
Because it needs to be specified for tech plugins that are separate from main Hammer. Since this is `example-tools.yml` you could just add these fields with filler text instead of ASAP7.